drivers/sqljs-driver/build.gradle

```diff
@@ -0,0 +1,79 @@
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
+
+kotlin {
+
+  js {
+    useCommonJs()
+    nodejs {
+      testTask {
+        useMocha()
+      }
+    }
+    browser {
+      testTask {
+        useMocha()
+      }
+    }
+  }
+
+  sourceSets {
+    commonMain {
+      dependencies {
+        implementation deps.kotlin.stdlib.common
+        implementation deps.kotlin.coroutines.common
+        api project (':sqldelight-runtime')
+      }
+    }
+    commonTest {
+      dependencies {
+        implementation deps.kotlin.test.common
+      }
+    }
+    jsMain {
+      dependencies {
+        implementation deps.kotlin.coroutines.js
+        api npm(deps.sqljs, versions.sqljs)
+      }
+    }
+    jsTest {
+      dependencies {
+        //implementation project(':drivers:driver-test')
+        implementation deps.kotlin.test.js
+      }
+    }
+  }
+}
+
+apply from: "$rootDir/gradle/gradle-mvn-mpp-push.gradle"
+
+publishing {
+  publications.all {
+    // Rewrite all artifacts from using the project name to just 'runtime'.
+    artifactId = artifactId.replace(project.name, 'sqljs-driver')
+  }
+}
+
+afterEvaluate {
+  // Alias the task names we use elsewhere to the new task names.
+  tasks.register('install') {
+    dependsOn('publishKotlinMultiplatformPublicationToMavenLocal')
+  }
+  tasks.register('installLocally') {
+    dependsOn 'publishMetadataPublicationToTestRepository'
+    dependsOn 'publishKotlinMultiplatformPublicationToTestRepository'
+    dependsOn 'publishJsPublicationToTestRepository'
+  }
+  // NOTE: We do not alias uploadArchives because CI runs it on Linux and we only want to run it on Mac OS.
+  //tasks.register('uploadArchives').dependsOn('publishKotlinMultiplatformPublicationToMavenRepository')
```

Comment:
Don't think we need this commented out since it should work on all host OSs.

Reply:
I removed all publishing related code for now, as per [Alec's comment](https://github.com/cashapp/sqldelight/pull/1486#issuecomment-564625921)